rust/tools/authorization-logic/src/souffle/souffle_interface.rs

```diff
@@ -40,6 +40,7 @@ pub fn emit_souffle(filename: &str) {
 /// translates it to datalogIR, and emits souffle code to a new file with the
 /// same name as the input but with the .dl extension in the out_dir
 pub fn input_to_souffle_file(filename: &str, in_dir: &str, out_dir: &str) {
+    println!("{}", &format!("indir: {}/{}", in_dir, filename));
```

Comment:
The use of `format!` here is redundant, as `println!` already accepts `format!`-style formatting for its first argument. Also, is this supposed to be here, or is this a leftover debug print>

Reply:
Thanks for catching that :) . This was not meant to be here, and it's written in that redundant way because I wanted to just lazily copy/paste it while debugging. It's gone now.